presto-product-tests/src/main/resources/sql-tests/testcases/table_with_header.sql

```diff
@@ -0,0 +1,11 @@
+-- database: presto; tables: table_with_header; groups: hive;
+-- delimiter: |; ignoreOrder: true;
+--! name: simple_scan
+SELECT count(*) FROM table_with_header
+--!
+32768
```

Comment:
Is it some tempto-ish way of specifying the expected value?
why there isn't the expected value for the next query? How do we verify it didn't read garbage?

Reply:
next query expects empty results, while first query expect single row with single column with `32768`